src/util/get-config-files.ts

```diff
@@ -32,11 +32,13 @@ export async function getConfigFiles(
     ref: branch,
   });
 
+  const files = [requestedRepoFile];
+
   // if no configuration file present in selected repository,
   // try to load it from the `.github` repository
-  if (!requestedRepoFile.config) {
+  if (!files[files.length - 1].config) {
```

Comment:
No need to change this line it is confusing to read.

Reply:
The reason why I changed it is to make clear that now the `files` array is the primary thing to work with.

Reply:
I am doing this to not mix variable anymore, so if someone comes with another code change above these lines here all it needs to be done is push to files, without the need of thinking about a second variable. I think it's cleaner.